docs/builders/iso.mdx

```diff
@@ -29,9 +29,12 @@ There are many configuration options available for the builder. They are
 segmented below into two categories: required and optional parameters. Within
 each category, the available configuration keys are alphabetized.
 
-In addition to the options listed here, a
-[communicator](/packer/docs/templates/legacy_json_templates/communicator) can be configured for this
-builder.
+In addition to the items listed here, you will want to look at the general
+configuration references for
+[VirtIO RNG device](#virtio-rng-device),
+[PCI Devices](#pci-devices),
+and [Communicator](/packer/docs/templates/legacy_json_templates/communicator)
+configuration references, which can be found further down the page.
```

Comment:
I may misunderstand something, but this section points to the external resources, so this feels a bit out of place imho. I would suggest leaving it as-is since it points out to the communicator docs outside of the plugin, while the PCI Devices/VirtIO Rng are sections within this plugin.

Reply:
Sorry, I don't quite follow. Are you saying to revert this change entirely or just keep the communicator in a seperate paragraph?

Reply:
Oh I meant only for the communicator here, since it is a different page I think it makes sense to keep a reference to it, it's just that mixing in anchors to sections of the current page seems out-of-place to me.

Reply:
Cool, yep that makes sense 👍